src/wxProfilerGUI/processlist.cpp

```diff
@@ -255,6 +266,14 @@ void ProcessList::sortByTotalCpuTime()
 		std::stable_sort(processes.begin(), processes.end(), TotalCpuTimeDescPred());
 }
 
+void ProcessList::sortByUpTime()
+{
+	if (sort_dir == SORT_UP)
+		std::stable_sort(processes.begin(), processes.end(), UpTimeAscPred());
+	else
+		std::stable_sort(processes.begin(), processes.end(), UpTimeDescPred());

```

Comment:
Could this (and `UpTimeDescPred`) be simplified using `processes.rbegin()` and `processes.rend()` ?